src/components/Common/common.css

```diff
@@ -1,3 +1,46 @@
 p {
-    @apply lil-font-medium lil-text-font-200;
+  font-size: 20px;
+  @apply lil-font-medium lil-text-font-200;
+}
+
+h1,
+h2,
+h3,
+h4,
+h5,
+h6 {
+  @apply lil-text-font-500;
+}
```

Comment:
styling already defined in text.css...... u can move these changes there....even for `p` tags